igor-web/src/main/java/com/netflix/spinnaker/igor/travis/service/TravisService.java

```diff
@@ -71,7 +71,8 @@
 
 public class TravisService implements BuildOperations, BuildProperties {
 
-  static final int TRAVIS_BUILD_RESULT_LIMIT = 100;
+  static final int TRAVIS_JOB_RESULT_LIMIT = 100;
+  static final int TRAVIS_BUILD_RESULT_LIMIT = 10;
```

Comment:
So I've renamed this variable everywhere _except_ where we want to limit the number of builds returned. So the code that is actually affected isn't really showing up in the changes below.